inception-project/inception-project.github.io
# Bugfix/18 query examples on search example do not work

**How to test**
* build site locally using `bundle exec jekyll serve --config _config.yml,_config_dev.yml` (cf. [docu](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/))
* access the local site with the browser
* import project to see if its ok 
* try copy/pasting queries from example description from the locally generated site into the search sidebar on the annotation page
* try copy/pasting queries from project description that appears in the INCEpTION UI into the search sidebar on the annotation page
